src/common/checksum.c

```diff
@@ -299,17 +357,42 @@ daos_csummer_finish(struct daos_csummer *obj)
 }
 
 bool
-daos_csummer_compare(struct daos_csummer *obj, daos_csum_buf_t *a,
-		     daos_csum_buf_t *b)
+daos_csummer_dcb_compare(struct daos_csummer *obj, daos_csum_buf_t *a,
```

Comment:
dcb == DAOS Checksum Buffer? it's not quite obvious and I suggest to just call it csbuf. In addition, in our code convention, we should use prefix_verb_postifx format for function name, so it's better to call it daos_csummer_cmp_csbuf

Reply:
yes, dcb == DAOS Checksum Buffer. There's a bunch of functions in checksum.h that use that acronym as the function prefix and I'm inclined to leave it. Are you okay if I change this function to daos_csummer_compare_dcb?